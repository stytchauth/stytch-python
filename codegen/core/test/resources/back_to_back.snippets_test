value = 123
# MANUAL(snippet1)
old_snippet1
# ENDMANUAL(snippet1)
# MANUAL(snippet2)
# ENDMANUAL(snippet2)
value2 = 456
