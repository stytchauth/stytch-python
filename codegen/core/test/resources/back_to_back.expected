value = 123
# MANUAL(snippet1)
hello world
# ENDMANUAL(snippet1)
# MANUAL(snippet2)
goodbye
# ENDMANUAL(snippet2)
value2 = 456
